app/lib/account/models.dart

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:gcloud/db.dart' as db;
+
+/// User data model.
+@db.Kind(name: 'User', idType: db.IdType.String)
+class User extends db.ExpandoModel {
+  String get userId => id as String;
+
+  @db.StringProperty()
+  String authProvider;
```

Comment:
If we want this, we need documentation.
I suggest that we don't do this... we should only have one auth provider... and keep it simple...

If we decide to add a different auth provider in the future, then we'll do a data migration and create an entity of kind `user_1` or something else :) hehe
(note the kind name is just in the annotation)

Reply:
removed